packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1006,6 +1007,18 @@ export class GarbageCollector implements IGarbageCollector {
 			return;
 		}
 
+		if (!toNodePath.startsWith("/")) {
+			// A long time ago we stored handles with relatives paths. We don't expect to see these cases though
+			// because GC was disabled after we made the switch to always using relative paths.
```

Comment:
Is this what you meant?
```suggestion
			// because GC was enabled only after we made the switch to always using absolute paths.
```

Reply:
Yes...